x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/crawl_requests_table.test.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { setMockValues } from '../../../../__mocks__/kea_logic';
+import '../../../__mocks__/engine_logic.mock';
+
+import React from 'react';
+
+import { shallow, ShallowWrapper } from 'enzyme';
+
+import { EuiBasicTable, EuiEmptyPrompt } from '@elastic/eui';
+
+import { mountWithIntl } from '../../../../test_helpers';
+
+import {
+  CrawlerDomain,
+  CrawlerPolicies,
+  CrawlerRules,
+  CrawlerStatus,
+  CrawlRequest,
+} from '../types';
+
+import { CrawlRequestsTable } from './crawl_requests_table';
+
+const values: { domains: CrawlerDomain[]; crawlRequests: CrawlRequest[] } = {
```

Comment:
[not a blocker, just curious] Is this type def necessary (i.e. typescript errors w/o it) or is it more for developer documentation?